thunder/tests/opinfos.py

```diff
@@ -347,6 +347,7 @@ def __init__(
         singularity_fn_producer=None,
         test_torch_compile_executor=False,
         instantiate_complex_tests=False,
+        no_fallback_with_double_inputs=False,
```

Comment:
It would be useful to have some documentation for this flag. What does it do, and why is it necessary for tests?

Reply:
I rewrote the reference implementation and removed this flag completely.